### PR TITLE
[Doc] Proofread Getting started

### PR DESCRIPTION
**Pull request by Vissidarte-Herman**
Related issue = #

### CODE REVIEW

File: docs/lang/articles/get-started/index.md [hunk 1]
```diff
@@ -5,7 +5,40 @@ slug: /
...
 # Getting Started
 
-Welcome to the Taichi Language documentation!
+## Target audience
+
+End users who *only* wish to quickly set up Taichi for simulation or high-performance numerical computation.
+
+:::caution IMPORTANT
+
+For developers who are interested in the compiler, computer graphics, or high-performance computing, and would like to contribute new features or bug fixes to the [Taichi programming language](https://github.com/taichi-dev/taichi), see the [Developer installation](../contribution/dev_install.md) for more information on building Taichi from source. Doing both at the same time may cause unnecessary conflicts.
```
Comment (ailzhang) [inline docs/lang/articles/get-started/index.md:14]: the sentence about "doing both" is outdated.

File: docs/lang/articles/get-started/index.md [hunk 2]
```diff
@@ -95,127 +124,95 @@ Let's dive into this simple Taichi program.
...
-|    macOS     |   OK    |   N/A    |    N/A     |    OK     |     N/A      |
-
-(OK: supported; N/A: not available)
-
-With `arch=ti.gpu`, Taichi will first try to run with CUDA. If CUDA is
-not supported on your machine, Taichi will fall back on Metal or OpenGL.
-If no GPU backend (CUDA, Metal, or OpenGL) is supported, Taichi will
-fall back on CPUs.
-:::
+- With `arch=ti.gpu`, Taichi first tries to run on CUDA. If CUDA is not supported on your machine, Taichi falls back on Metal or OpenGL.
```
Comment (ailzhang) [inline docs/lang/articles/get-started/index.md:143]: see above.

**Review by ailzhang — APPROVED** [review]
LGTM with minor nits!

File: docs/lang/articles/get-started/index.md (hunk 1 above)
Comment (Vissidarte-Herman) [reply docs/lang/articles/get-started/index.md:14]: Will also move this from dev install.

File: docs/lang/articles/get-started/index.md [hunk 3]
```diff
@@ -5,7 +5,40 @@ slug: /
...
+
+Taichi recommends installing Python from [Miniforge](https://github.com/conda-forge/miniforge/#download) conda if you are on a MacBook with M1 chip.
+
+:::
+
+### Supported systems and backends
+
+The following table lists the supported operating systems and the backends that Taichi supports on these platforms:
+
+| **platform** |      **CPU**       |      **CUDA**      |     **OpenGL**     |     **Metal**      |    **C source**    |
```
Comment (strongoier) [inline docs/lang/articles/get-started/index.md:34]: Should we add Vulkan and remove C source here?

Comment (turbo0628) [reply docs/lang/articles/get-started/index.md:34]: Agree, the Vulkan backend works smoothly.

**Review by rexwangcc — APPROVED** [review]
LGTM!

File: docs/lang/articles/get-started/index.md [hunk 4]
```diff
@@ -288,14 +269,13 @@ def bar(k: ti.i32):
...
+:::caution WARNING
 
-`break` **is not supported in parallel loops**:
+`break` is *not* supported in parallel loops:
 
-```python {5,9,16}
+```python
 @ti.kernel
 def foo():
   for i in x:
```
Comment (strongoier) [inline docs/lang/articles/get-started/index.md:281]: Should 4 spaces be used for indentation here and in the GUI system section?

**Review by strongoier — APPROVED** [review]

**Pull request merged**